src/overlays/kaleido_scope/ovl_kaleido_scope/z_kaleido_scope_NES.c

```diff
@@ -242,9 +237,12 @@ s16 sPauseCursorRightX = 0;
 
 s16 D_8082B920 = 10;
 
-u8 D_8082B924[] = {
-    0x00, 0x14, 0x00, 0x04, 0x00, 0x14, 0x00, 0x0A, 0x0F, 0x1F, 0x0F, 0x1F, 0x1F, 0x1F, 0x1F, 0x1F,
-    0x07, 0x07, 0x03, 0x0F, 0x07, 0x0F, 0x0F, 0xFF, 0xFF, 0xFF, 0x1F, 0x0F, 0x03, 0x0F, 0x00, 0x00,
+s16 D_8082B924[] = { 20, 4, 20, 10 };
```

Comment:
Only `D_8082B924[0]` seems to get used (as the cursor color timer so name suggestion is `sCursorColorTimer`)? Is it possible to actually split this up, and have the `4, 20, 10` as a different array?

Reply:
Take a look at `s16 sQuestHpColorTimerInit[] = { 20, 4, 20, 10 };` which is used in the same way, except it's actually indexed instead of just indexing 0.

I think it make more sense as 4 since `sPauseLRCursorColorTargets` also has 4 targets, and it almost feels like a bug to not use all 4 timers, but maybe it was intended. To parallel that, I would name it `sPauseLRCursorColorTimerInit`?

Reply:
I would only suggest adding an `s`, i.e Inits (for both I think).